### ISSUE TITLE
Error removing bookmark on server

### ISSUE THREAD

**Issue by Riccardo2017** [marcelklehr/floccus#102]
### Software versions
* Browser: Chrome
* Nextcloud: 13.0.2
* Nextcloud Bookmarks app: 0.11.0
* Floccus: 2.2.2

### Steps to reproduce
Start syncing

### Expected outcome
Sync should be finished without error messages.

### Actual outcome
I get the following error message: Removing a bookmark on the server failed. remoteId=1793 Removing a bookmark on the server failed. remoteId=1793

How can I identify with which bookmark Floccus has problems? With the remoteId 1793 I cannot search for the bookmark and find out why it can't be deleted.

**Comment by Riccardo2017** [comment]
One more information: I use Floccus for a few days now and are happy, because I used XMarks before and this service discontinued. Today, a few hours ago, Floccus tells me an error message, because one bookmark was stored locally 4 times. This was true, I've stored one bookmark in 4 different folders.
I decided to delete 3 bookmarks, so I have only 1 bookmark left. After that the sync process started and ended without error messages.

I am not sure if my error message have something todo with this case, but maybe it helps somebody to find a solution for my issue.
